vms/platformvm/validator.go

```diff
@@ -72,7 +72,7 @@ func (v *SubnetValidator) SubnetID() ids.ID { return v.Subnet }
 // Verify this validator is valid
 func (v *SubnetValidator) Verify() error {
 	switch {
-	case v.Subnet.IsZero():
+	case v.Subnet == ids.Empty:
```

Comment:
This wasn't as intended, but I'm not sure we should remove it.... (although if we want to keep it, we should change ids.Empty to `constants.PrimarySubnetID`